src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -78,6 +78,12 @@ func validateConfig(_cfg yamlconf.Config) error {
 	if !ok {
 		return fmt.Errorf("Config is not a flashlight config!")
 	}
+	if cfg.Client == nil {
+		return fmt.Errorf("No client config")
+	}
+	if cfg.Client.ChainedServers == nil {
+		return fmt.Errorf("No chained servers")
+	}
```

Comment:
This actually just knocks out https://github.com/getlantern/lantern/issues/4357 while we're in here.